frontend/controller/actions/group.js

```diff
@@ -112,10 +149,36 @@ export default (sbp('sbp/selectors/register', {
     try {
       // post acceptance event to the group contract
       const message = await sbp('chelonia/out/actionEncrypted', {
-        action: 'gi.contracts/group/inviteAccept', ...params
+        ...omit(params, ['options']),
+        action: 'gi.contracts/group/inviteAccept',
+        hooks: {
+          prepublish: params.hooks?.prepublish,
+          postpublish: null
+        }
       })
       // sync the group's contract state
       await sbp('state/enqueueContractSync', params.contractID)
+
+      // join the 'General' chatroom by default
+      const rootState = sbp('state/vuex/state')
+      const generalChatRoomId = rootState[params.contractID].generalChatRoomId
+      if (generalChatRoomId) {
+        await sbp('gi.actions/group/joinChatRoom', {
+          ...omit(params, ['options']),
+          data: {
+            chatRoomID: generalChatRoomId
+          },
+          hooks: {
+            prepublish: null,
+            postpublish: params.hooks?.postpublish
+          }
+        })
+      } else {
+        throw new GIErrorUIRuntimeError(L('Failed to join the group: {codeError}', {
+          codeError: 'Failed in joining General chatroom. This won\'t be happened.'
+        }))
```

Comment:
1. It's important to pass down the original error, so like elsewhere, `codeError` should be set to `e.message`
2. The error message must reflect what happened. Here, we didn't `Failed to join the group`. The message should be: `L('Failed to join general chatroom: {codeError}'`

Reply:
And generally speaking it's better to use to use `...LError(e)` with `{reportError}` in the error string. See for example how `'gi.actions/group/removeMember'` works

Reply:
1. Make sense. But I don't think we could have that `e` and could use `e.message` here.
2. Updated like the following.
```
throw new GIErrorUIRuntimeError(L('Failed to join the group: {reportError}', {
  reportError: `No ${CHATROOM_GENERAL_NAME} chatroom is there in the group.`
}))
```